### DISCUSSION TITLE
Allowing A Player To Name Their Character

### DISCUSSION THREAD

**Discussion by egyptthecreator** [inkle/ink#914] [Writing in ink]
I want to allow a player to pick their name for the duration of a playthrough.

Is there a way to create a text box that flags the player's input as a variable to recall throughout the story?

**Comment by premek** [comment]
you can display the text box in your game engine and then set the value into an ink variable, see 

https://github.com/inkle/ink/blob/master/Documentation/RunningYourInk.md#settinggetting-ink-variables
